crates/net/network/src/manager.rs

```diff
@@ -196,9 +197,20 @@ where
             disc_config
         });
 
+        #[cfg(not(feature = "discv5"))]
```

Comment:
we can also wrap discv5 in an option and skip the feature, that creates breaking changes in publicly exported types in the network crate, so I tried to avoid it